### PR TITLE
Feature enter critical ratio

### PR DESCRIPTION
**Pull request by christopher77**
**Status**
  Ready

**Implementation Notes**

* Add Critical-Ratio view
* Add test to Critical-ratio
* Add myCriticalRatio and overbookingNumber to InitialState

### CODE REVIEW

File: frontend/src/views/critical-ratio.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        autoComplete="off"
+        type="number"
+        placeholder="enter the critical ratio"
+        onChange={onChangeInputCriticalRatio}
+        value={criticalRatio}
+      />
+      <Button type="submit" disabled={!isEnabled}>
+        Set Critical Ratio
+      </Button>
+      {/* <button  css={buttoncomponent}>
```
Comment (valevassallo) [inline frontend/src/views/critical-ratio.js]: Please get rid of comments beforehand

Comment (christopher77) [reply frontend/src/views/critical-ratio.js]: ok, I forgot to delete it,thanks

File: frontend/src/views/critical-ratio.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+function CriticalRatio() {
+  const [criticalRatio, setCriticalRatio] = React.useState("");
+  const [isEnabled, setIsEnabled] = React.useState(false);
+
+  const addCriticalRatio = useAddCriticalRatio();
+
+  const state = useRound();
+
+  const buttoncomponent = {
```
Comment (yummta) [inline frontend/src/views/critical-ratio.js]: I suggest using camelCase to all variables' name.

**Review by yummta — APPROVED** [review]

File: frontend/src/views/critical-ratio.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    padding: ".5rem",
+    color: "#34495e",
+    width: "100%",
+    "&:focus": {
+      outline: "none",
+      borderColor: "rgba(242, 107, 117, 0.5)"
+    }
+  };
+
+  const contenedor = {
```
Comment (yummta) [inline frontend/src/views/critical-ratio.js]: What do you think about using only English for variables' names?

File: frontend/src/views/critical-ratio.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  function handleSubmit(event) {
+    event.preventDefault();
+    addCriticalRatio(criticalRatio);
+  }
+  return (
+    <form css={contenedor} onSubmit={handleSubmit}>
+      <img css={imagecss} src={plane} alt="plane" />
+      <div css={labelcss}>Flight PDR456</div>
+      <div css={databox}>
+        <div css={fieldset}>
```
Comment (yummta) [inline frontend/src/views/critical-ratio.js]: This element contains relevant text. What do you think about using others tags more semantic like <p>, <span>, <b>, etc. 

Maybe: 
```
        <p css={fieldset}>
          Underage Cost<b>{state.underageCost}</b>
        </p>
```

File: frontend/src/views/critical-ratio.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+          Cost of Overbooking<p>{state.overbookingCost}</p>
+        </div>
+        <div css={fieldset}>
+          Underage Cost<p>{state.underageCost}</p>
+        </div>
+      </div>
+      <div css={labelcss}>Calculate the Critical Ratio</div>
```
Comment (yummta) [inline frontend/src/views/critical-ratio.js]: This element is the label from the next input. So, I think that this `<div>` should be a `<label>`tag. What about you think about it?

**Review by yummta — DISMISSED** [review]

File: frontend/src/views/critical-ratio.js (hunk 2 above)
Comment (christopher77) [reply frontend/src/views/critical-ratio.js]: yeah, you're right.

File: frontend/src/views/critical-ratio.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+import React from "react";
+import { jsx } from "@emotion/core";
+import plane from "../images/plane.png";
+import { useRound } from "../selectors";
+import { useAddCriticalRatio } from "../action-hooks";
+import { Button } from "../components/ui";
+
+function CriticalRatio() {
+  const [criticalRatio, setCriticalRatio] = React.useState("");
+  const [isEnabled, setIsEnabled] = React.useState(false);
```
Comment (valevassallo) [inline frontend/src/views/critical-ratio.js:11]: I really like this, good idea!

File: frontend/src/views/critical-ratio.js (hunk 3 above)
Comment (christopher77) [reply frontend/src/views/critical-ratio.js]: I forgot to delete it too, yes I will change it

**Review by valevassallo — DISMISSED** [review]

File: frontend/src/views/critical-ratio.js (hunk 6 above)
Comment (christopher77) [reply frontend/src/views/critical-ratio.js:11]: thanks

File: frontend/src/views/critical-ratio.js (hunk 5 above)
Comment (christopher77) [reply frontend/src/views/critical-ratio.js]: Yes, I think I will change it

**Review by valevassallo — APPROVED** [review]

**Review by yummta — APPROVED** [review]

**Pull request merged**